src/Microsoft.PowerShell.ScheduledJob/ScheduledJobDefinition.cs

```diff
@@ -20,7 +20,6 @@ namespace Microsoft.PowerShell.ScheduledJob
     /// can be scheduled to run through either stand-alone or through the Windows
     /// Task Scheduler.
     /// </summary>
-    [Serializable]
```

Comment:
This breaks Scheduled Jobs functionality.  Should we add `Obsolete` attribute to the cmdlets?

Reply:
Scheduled job module is excluded from our build (we don't ship scheduled job module), so we should not make changes to its code.